_docs/reference/release-roadmap.md

```diff
@@ -15,6 +15,7 @@ Istio 0.2 will be our second release, and will work across multiple Kubernetes c
 - Support for VMs joining an Istio Mesh.
 - Fully-automated injection of Envoy proxies into Kubernetes pods.
 - Standard extension points for addons and plugins.
+- Expanded service-to-service authentication support for VMs.
```

Comment:
We already have "Support for VMs joining an Istio Mesh". What does this add?

That item already includes service naming, authn, authz, ....